setup.cfg

```
@@ -22,11 +22,13 @@ classifiers =
     Programming Language :: Python :: Implementation :: CPython
     Programming Language :: Python :: Implementation :: PyPy
     Topic :: System :: Systems Administration
+    Typing :: Typed
 
 [options]
 python_requires = >=3.7
 setup_requires = setuptools_scm
 packages = find:
+include_package_data = true
```

```suggestion
package_data = piptools = py.typed
```

Mayby explicitly define certain package data?